hydromt_fiat/workflows/exposure_vector.py

```diff
@@ -310,13 +310,14 @@ def setup_buildings_from_multiple_sources(
         max_dist: Union[int, float,List[float], List[int], None] = 10,
         ground_elevation_file: Union[int, float, str, Path, None] = None,
         ground_elevation_unit: str = None,
+        bf_conversion: bool = False
     ):
         self.logger.info("Setting up exposure data from multiple sources...")
         self.setup_asset_locations(asset_locations)
         self.setup_occupancy_type(occupancy_source, occupancy_attr)
         self.setup_max_potential_damage(max_potential_damage, damage_types, country = country)
         if any(isinstance(geom, Polygon) for geom in self.exposure_geoms[0]['geometry']):
-            self.convert_bf_into_centroids(self.exposure_geoms[0], self.exposure_geoms[0].crs)
+            self.convert_bf_into_centroids(self.exposure_geoms[0], self.exposure_geoms[0].crs, bf_conversion)
```

Comment:
I think it would be better to check if bf_conversion is true in the if statement and only then call the function.
The function (if called) should always do the conversion (so you should get rid of the bf_conversion argument:

`if any(isinstance(geom, Polygon) for geom in self.exposure_geoms[0]['geometry']) and bf_conversion`

Reply:
fixed :)